File: src/2-Problem05.cpp
#include <iostream>
using namespace std;

int main () {
    // Kamus, asumsi jumlah anggota kelompok < 999 dan tidak negatif
    int jlhkelompok, anggotakelompok, minanggota = 999, nokelompok;
    // Masukkan jumlah kelompok
    cout << "Masukan jumlah kelompok :";
    cin >> jlhkelompok;
    // Looping, menentukan jumlah anggota minimum seiring dengan dimasukkannya input pengguna
    for (int i = 1; i <= jlhkelompok; i++) {
        cout << "Masukan jumlah anggota kelompok " << i << " :";
        cin >> anggotakelompok;
        if (anggotakelompok < minanggota) {
            nokelompok = i;
            minanggota = anggotakelompok;
        }
    }
    // Tampilkan kelompok dengan jumlah anggota minimum
    cout << "Kelompok dengan anggota tersedikit adalah kelompok " << nokelompok << endl;
    return 0;
}

File: src/2-Problem05.py
#Program mencari index nilai terkecil

numKelompok = int(input('Masukan jumlah kelompok :'))
minAnggota = None
kelMin = None
tempNumAnggota = None

for i in range(numKelompok):
  tempNumAnggota = int(input('Masukan jumlah anggota kelompok {} :'.format(i + 1)))
  if(kelMin is None):
    kelMin = i + 1
    minAnggota = tempNumAnggota
  else : 
    if(tempNumAnggota < minAnggota):
      minAnggota = tempNumAnggota
      kelMin = i + 1

print('Kelompok dengan anggota tersedikit adalah kelompok {}'.format(kelMin))


